R/phsproject.R

```diff
@@ -81,11 +81,7 @@ phsproject <- function(path, author, n_scripts = 1, git = FALSE, renv = FALSE, o
   }
 
   if (git) {
-    if (Sys.info()[["sysname"]] == "Windows") {
-      shell(paste("cd", path, "&&", "git init"))
-    } else {
-      system(paste("cd", path, "&&", "git init"))
-    }
+    git2r::init(file.path(getwd(), path))
```

Comment:
Why not also make an initial commit here (the `session = TRUE` isn't required but I think is  a nice touch).
```suggestion
    git2r::init(file.path(getwd(), path))
    
    git2r::commit(message = "Initial commit", all = TRUE, session = TRUE)
```

Same goes for the Shiny App (possibly more so).